module/Photo/view/photo/album/index-new.phtml

```diff
@@ -0,0 +1,265 @@
+                ?>
+                <a href="<?= $this->url('photo/year', ['year' => $associationYear->getYear()]); ?>">
+                    <?= $associationYear->getYearString() ?>
+                </a>
+            </li>
+            <?php endif; ?>
+            <?php
+            // first we need to get all parents
+            $crumbs = [];
+            if (!is_null($album->getParent())) {
+                $currentAlbum = $album;
+                while(!is_null($currentAlbum->getParent())) {
+                    // prepend parent to array
+                    array_unshift($crumbs, [
+                        'url'  => $this->url('photo/album', ['album_id' => $currentAlbum->getParent()->getId()]),
+                        'name' => $currentAlbum->getParent()->getName()
+                    ]);
+                    $currentAlbum = $currentAlbum->getParent();
+                }
+            }
+            // then we print them
+            foreach($crumbs as $crumb): ?>
+                <li>
+                    <a href="<?= $crumb['url'] ?>">
+                        <?= $this->escapeHtml($crumb['name']) ?>
+                    </a>
+                </li>
+            <?php endforeach; ?>
+            <li class="active">
+                <?php if ($album instanceof \Photo\Model\MemberAlbum): ?>
+                    <span class="glyphicon glyphicon-user"></span>
+                    &nbsp;
+                <?php endif; ?>
+                <?= $this->escapeHtml($album->getName()); ?>
+            </li>
+        </ol>
+    </div>
+</section>
+
+<section class="section">
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">
+                    <?= $this->escapeHtml($album->getName()); ?>
+                    <small>
+                        <?php if ($album instanceof \Photo\Model\MemberAlbum): ?>
+                            <a href="<?= $this->url('member/view', ['lidnr' => $album->getMember()->getLidnr()]) ?>">
+                                <?= strtolower($this->translate("View user profile")) ?>
+                            </a>
+                        <?php else: ?>
+                            <?= $album->getStartDateTime()->format('d-m-Y') ?>
+                        <?php endif; ?>
+                    </small>
+                </h1>
+            </div>
+        </div>
+        <?php
+        $path = '';
+        if ($album instanceof \Photo\Model\MemberAlbum) {
+            $path = $this->url('photo/member', ['lidnr' => $album->getMember()->getLidnr()]);
+        } else {
+            $path = $this->url('photo/album', ['album_id' => $album->getId()]);
+        }
+        ?>
+
+        <div class="photo-grid">
+            <div class="grid-sizer"></div>
+            <div class="gutter-sizer"></div>
+            <?php foreach ($album->getChildren() as $item): ?>
+                    <!-- TODO: fix this -->
```

Comment:
What needs to be fixed? Is it important for deploying a beta version?

Reply:
This will be fixed by issues #926, therefore I didn't fix it here. So it is not important for the beta.